test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2617,6 +2617,65 @@ def forward(self, *tensor_list):
         x = torch.randn(3, 4)
         self.run_test(EinsumModelTranspose(), input=(x,))
 
+    @skipIfUnsupportedMinOpsetVersion(12)
```

Comment:
also temporarly add something like : @unittest.skip("Enable this once ORT version is updated")